codes/c/chapter_sorting/radix_sort.c

```diff
@@ -36,16 +37,17 @@ void countingSortDigit(int nums[], int size, int exp) {
         counter[d]--;           // 将 d 的数量减 1
     }
     // 使用结果覆盖原数组 nums
-    for (int i = 0; i < size; i++) {
-        nums[i] = res[i];
-    }
+    memcpy(nums, res, size * sizeof(int));
```

Comment:
Don't use a specific api, it's better to be consistent with other language versions([cpp](https://github.com/krahets/hello-algo/blob/main/codes/cpp/chapter_sorting/radix_sort.cpp#L38)、[go](https://github.com/krahets/hello-algo/blob/main/codes/go/chapter_sorting/radix_sort.go#L38))

Reply:
hi @MultiWolf, could you please address this comment?

Reply:
> Okey, Resolved. In the most recent commit, revert to using the old loop override array method and drop specific api's like memcpy.

@Gonglja @krahets Resolved, please review again, thenks!